frontend/src/app/main/data/workspace/colors.cljs

```diff
@@ -821,39 +823,43 @@
     (update [_ state]
       (update state :colorpicker
               (fn [{:keys [stops editing-stop] :as state}]
-                (if (cc/uniform-spread? stops)
-                  ;; Add to uniform
-                  (let [stops (->> (cc/uniform-spread (first stops) (last stops) (inc (count stops)))
-                                   (mapv split-color-components))]
-                    (-> state
-                        (assoc :current-color (get stops editing-stop))
-                        (assoc :stops stops)))
-
-                  ;; We add the stop to the middle point between the selected
-                  ;; and the next one.
-                  ;; If the last stop is selected then it's added between the
-                  ;; last two stops.
-                  (let [index
-                        (if (= editing-stop (dec (count stops)))
-                          (dec editing-stop)
-                          editing-stop)
-
-                        {from-offset :offset} (get stops index)
-                        {to-offset :offset}   (get stops (inc index))
-
-                        half-point-offset
-                        (+ from-offset (/ (- to-offset from-offset) 2))
-
-                        new-stop (-> (cc/interpolate-gradient stops half-point-offset)
-                                     (split-color-components))
-
-                        stops (conj stops new-stop)
-                        stops (into [] (sort-by :offset stops))
-                        editing-stop (d/index-of-pred stops #(= new-stop %))]
-                    (-> state
-                        (assoc :editing-stop editing-stop)
-                        (assoc :current-color (get stops editing-stop))
-                        (assoc :stops stops)))))))))
+                (let [cap-stops? (or (features/active-feature? state "render-wasm/v1") (contains? cfg/flags :binary-fills))
+                      can-add-stop? (if cap-stops? (< (count stops) shp/MAX-GRADIENT-STOPS) true)]
```

Comment:
you can change this to `(or (not cap-stops?) (< (count stops) shp/MAX-GRADIENT-STOPS))`